classes/eztagstreeattributefilter.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * eZTagsTreeAttributeFilter class implements TagsTreeAttributeFilter extended attribute
+ *
+ */
+class eZTagsTreeAttributeFilter
+{
+    /**
+     * Creates and returns SQL parts used in fetch functions
+     *
+     * @return array
+     */
+    function createSqlParts( $params )
+    {
+        $returnArray = array( 'tables' => '', 'joins'  => '', 'columns' => '' );
+        
+        if ( isset( $params['parent_tag_id'] ) )
+        {
+            if ( is_array( $params['parent_tag_id'] ) )
+            {
+                $tagIDArray = $params['parent_tag_id'];
+            }
+            else if ( is_string( $params['parent_tag_id'] ) )
```

Comment:
Now that parent tag ID can be an integer, this will break if someone passes an int here.

I propose using the same check as in other attribute filter: https://github.com/ezsystems/eztags/blob/master/classes/eztagsattributefilter.php#L19-L33